docs/user-guide/networkpolicies.md

```diff
@@ -64,6 +64,9 @@ spec:
     role: db
  ingress:
   - from:
+     - namespaceSelector:
```

Comment:
Yep, YAML strikes again, but they are in separate Peers so this should be OK.

@eparis but while we're here, would be nice to have an explanation below of what this policy is actually doing.  For example:

```
In plain language, this NetworkPolicy:
1. applies to all pods in the current (???) namespace with the label "role=db"
2. allows tcp/6379 ingress traffic to the "role=db" pods from any pod in the current namespace with the label "role=frontend" (due to the podSelector list element)
3. allows tcp/6379 ingress traffic to the "role=db" pods from any pod in the namespace "myproject" (due to the namespaceSelector list element)
```